tv_apps/browser/js/search_util.js

```diff
@@ -0,0 +1,283 @@
+'use strict';
+
+var SearchUtil = {
+  // country code
+  COUNTRY: 'US',
+
+  /* search engine list of the search bar */
+  ENGINE_LIST: {
+    'CN': [
+      {id: 2, name: 'Baidu'},
+      {id: 1, name: 'Bing'}
+    ],
+    // default
+    'US': [
+      {id: 1, name: 'Bing'},
+      {id: 0, name: 'Google'}
+    ]
+  },
+  // search engine
+  engineList: null,
+  searchId: null,
+  searchIdx: null,
+
+  DOMAIN_LIST: {
+    'US':'com'    , 'CA':'ca'     , 'DE':'de'     , 'IE':'ie'     , 
+    'NL':'nl'     , 'HU':'hu'     , 'PL':'pl'     , 'CZ':'cz'     , 
+    'SK':'sk'     , 'IT':'it'     , 'GB':'co.uk'  , 'SI':'si'     , 
+    'GR':'gr'     , 'PT':'pt'     , 'DK':'dk'     , 'FI':'fi'     , 
+    'HR':'hr'     , 'NO':'no'     , 'RO':'ro'     , 'RU':'ru'     , 
+    'BG':'bg'     , 'LT':'lt'     , 'YU':'rs'     , 'SE':'se'     , 
+    'LV':'lv'     , 'EE':'ee'     , 'TR':'com.tr' , 'BR':'com.br' , 
+    'ID':'co.id'  , 'CN':'com.hk' , 'TW':'com.tw' , 'HK':'com.hk' , 
+    'VN':'com.vn' , 'TH':'co.th'  , 'SA':'com.sa' , 'FR':'fr'     , 
+    'ES':'es'     , 'SG':'com.sg' , 'AU':'com.au' , 'NZ':'co.nz'  , 
+    'IN':'co.in'  , 'MY':'com.my' , 'AE':'ae'     , 'KW':'com.kw' , 
+    'MX':'com.mx' , 'AT':'at'     , 'BE':'be'     , 'CH':'ch'     , 
+    'MT':'com.mt' , 'AD':':'      , 'LU':'lu'     , 'IS':'is'     , 
+    'UA':'com.ua' , 'KZ':'kz'     , 'CIS':'by'    , 'UK':'co.uk'  , 
+    'CL':'cl'     , 'PE':'com.pe' , 'CO':'com.co' , 'AR':'com.ar' , 
+    'PA':'com.pa' , 'JP':'co.jp'  , 'MK':'mk'     , 'MM':'com.mm'
+  },
+  // search domain
+  domainList: null,
+
+  /* URL of each search engine */
+  SEARCH_URL: {
+    //Google :
+    0 : [
+      "http://www.google.DOMAIN/search?ie=UTF-8&oe=UTF-8&q=",
+      "http://www.google.DOMAIN/search?ie=UTF-8&oe=UTF-8&tbm=isch&q=",
+      "http://www.google.DOMAIN/search?ie=UTF-8&oe=UTF-8&tbm=vid&q=",
+      "http://www.google.DOMAIN/search?ie=UTF-8&oe=UTF-8&tbm=nws&q=",
+      "http://maps.google.com/maps?q=",
+      "http://www.youtube.DOMAIN/results?search_query="
```

Comment:
These search URLs may need to be confirmed (and remove inappropriate one) before check-in.